editoast/src/views/round_trips.rs

```diff
@@ -38,6 +59,20 @@ pub(in crate::views) struct RoundTrips {
     round_trips: Vec<(i64, i64)>,
 }
 
+impl RoundTrips {
+    /// Check if it contains duplicate ids in both one-ways and round trips
+    fn has_duplicates(&self) -> bool {
+        // Using sort and dedup is faster than using a HashSet
```

Comment:
Wut? Why? Not enough elements typically, optimized to a single memcpy, something else? :o